src/main/java/org/jfrog/hudson/pipeline/common/types/deployers/GradleDeployer.java

```diff
@@ -185,4 +188,23 @@ public String getTargetRepository(String deployPath) {
                 .artifactoryPluginVersion(ActionableHelper.getArtifactoryPluginVersion())
                 .publications(getPublications().getPublications());
     }
+
+    /**
+     * Adds the provided artifacts to the Deployed Gradle Artifacts Summary Action.
+     * If such action was not initialized yet, initialize a new one.
+     */
+    public static void addDeployedGradleArtifactsToAction(Run build, List<DeployedArtifact> gradleArtifacts) {
+        if (gradleArtifacts.isEmpty()) {
+            return;
+        }
```

Comment:
This logic here and in addDeployedMavenArtifactsToAction can be moved up to addDeployedArtifactsToAction.
```suggestion
```